packages/apollo-server-plugin-response-cache/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.base",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./dist"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["**/__tests__", "**/__mocks__"],
+  "references": [
+    { "path": "../apollo-cache-control" },
+    { "path": "../apollo-server-plugin-base" },
+    { "path": "../apollo-server-caching" }
```

Comment:
I'm somewhat surprised we don't reference the `apollo-server-env` project here, but also noticing that we don't do that anywhere else. 🤷‍♂️ 	

Note: I don't think this means you should change anything, just a casual observation when I was mentally double-clicking on the dependencies.